src/components/actions/filters/ItemFilter.tsx

```diff
@@ -13,21 +15,25 @@ export const ELLIPSIS: string = '...';
 export class ItemFilter extends React.Component<IItemFilterProps, any> {
 
   render() {
-    let itemFilter: string = this.props.item;
+    let itemFilterText: string = this.props.item;
     if (this.props.crop) {
-      let itemFilterLength: number = itemFilter.length;
+      let itemFilterLength: number = itemFilterText.length;
 
       if (itemFilterLength > Math.abs(this.props.crop)) {
-        itemFilter = this.props.crop > 0
-          ? itemFilter.substring(0, this.props.crop) + ELLIPSIS
-          : ELLIPSIS + itemFilter.substring(itemFilterLength + this.props.crop, itemFilterLength);
+        itemFilterText = this.props.crop > 0
+          ? itemFilterText.substring(0, this.props.crop) + ELLIPSIS
+          : ELLIPSIS + itemFilterText.substring(itemFilterLength + this.props.crop, itemFilterLength);
       }
     }
 
+    const itemFilter: JSX.Element = this.props.itemTooltipProps
+      ? <Tooltip {...this.props.itemTooltipProps}><span className='item-filter-item'>{itemFilterText}</span></Tooltip>
```

Comment:
I'd put the span on it's own line so it's easier to see the html

Reply:
if the title in itemTooltipProps is empty or undefined, the Tooltip will wrap your item-filter with a span. I dont think you need this validation here. You can test but you can always use the Tooltip tag